File: launchdarkly/ruby/src/GetPullRequestsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::InsightsPullRequestsBetaApi.new.get_pull_requests(
        "projectKey_string", # project_key
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling InsightsPullRequestsBetaApi#get_pull_requests: #{e}"
end

File: launchdarkly/python/src/GetPullRequestsExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.InsightsPullRequestsBetaApi(api_client).get_pull_requests(
            project_key="projectKey_string",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling InsightsPullRequestsBetaApi#get_pull_requests: %s\n" % e)
